src/client/wire_protocol/operations.rs

```diff
@@ -0,0 +1,312 @@
+use bson::Document as BSON;
```

Comment:
Shoot, I must have misremembered the style guideline for that, for some reason I thought it was the other way around.